addons/l10n_ar/report/invoice_report_view.xml

```diff
@@ -7,6 +7,7 @@
         <field name="inherit_id" ref="account.view_account_invoice_report_search" />
         <field name="arch" type="xml">
             <search>
+                <field name="company_id"/>
```

Comment:
@Bruno-Zanotti groups="base.group_multi_company"  I don't know if it makes that much sense.  A group by company seems more logical.  @ren-odoo